cv_bridge/CMakeLists.txt

```diff
@@ -3,16 +3,18 @@ project(cv_bridge)
 
 find_package(catkin REQUIRED COMPONENTS rosconsole sensor_msgs)
 
+find_package(Boost QUIET)
 if(NOT ANDROID)
-  find_package(PythonLibs)
```

Comment:
The rationale behind removing this is the it gets called implicitly by `find_package(Boost QUIET)`?

Reply:
Oh, no it doesn't. I'm not sure why my testing worked without this, but it definitely should have been there still.